### PR TITLE
nextpnr-xilinx: add computation of needed data files

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

This PR adds the computation of the .bba and .bin files for nextpnr-xilinx

### CODE REVIEW

**Review by mithro — APPROVED** [review]
LGTM

**Comment by acomodi** [comment]
I get this error during the nextpnr-xilinx build:
```
+nextpnr-xilinx -V
nextpnr-xilinx: error while loading shared libraries: libboost_python36.so.1.72.0: cannot open shared object file: No such file or directory
```

This did not happen previously. Probably due to the addition of pypy3?

**Comment by acomodi** [comment]
@mithro the package now contains the required files:

```
bin/nextpnr-xilinx
info/about.json
info/files
info/git
info/hash_input.json
info/has_prefix
info/index.json
info/licenses/COPYING
info/paths.json
info/recipe/build.sh
info/recipe/conda_build_config.yaml
info/recipe/condarc
info/recipe/meta.yaml
info/recipe/meta.yaml.template
info/recipe/recipe_append.yaml
info/test/run_test.sh
share/nextpnr-xilinx/xc7a35tcsg324-1.bba
share/nextpnr-xilinx/xc7a35tcsg324-1.bin
```

**Comment by mithro** [comment]
@acomodi - Should there be xc7a200t and other similar files there too?

**Comment by acomodi** [comment]
@mithro Added the remaining devices. Unfortunately 200t cannot be used as the submodule to prjxray-db is pointing to an old version, that needs to change upstream.

File: nextpnr-xilinx/meta.yaml [hunk 1]
```diff
@@ -22,19 +22,22 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - python {{ python }}
+    - libboost
+    - py-boost
     - yosys
+    - pypy3.6
```
Comment (mithro) [inline nextpnr-xilinx/meta.yaml:28]: The dependency on `pypy3.6` worries me a little. Let's merge and log an issue to follow up on that.

**Comment by mithro** [comment]
The contents output are now;
```
bin/nextpnr-xilinx
info/about.json
info/files
info/git
info/hash_input.json
info/has_prefix
info/index.json
info/licenses/COPYING
info/paths.json
info/recipe/build.sh
info/recipe/conda_build_config.yaml
info/recipe/condarc
info/recipe/meta.yaml
info/recipe/meta.yaml.template
info/recipe/recipe_append.yaml
info/test/run_test.sh
share/nextpnr-xilinx/xc7a35tcpg236-1.bba
share/nextpnr-xilinx/xc7a35tcpg236-1.bin
share/nextpnr-xilinx/xc7a35tcsg324-1.bba
share/nextpnr-xilinx/xc7a35tcsg324-1.bin
share/nextpnr-xilinx/xc7z010clg400-1.bba
share/nextpnr-xilinx/xc7z010clg400-1.bin
share/nextpnr-xilinx/xc7z020clg484-1.bba
share/nextpnr-xilinx/xc7z020clg484-1.bin
```

**Pull request merged**